x-pack/plugins/license_api_guard/server/license.test.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { of } from 'rxjs';
+import type { KibanaRequest, RequestHandlerContext } from 'src/core/server';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { httpServerMock } from 'src/core/server/http/http_server.mocks';
```

Comment:
@mshustov Do you think we could update `src/core/server` to export these helpers publicly?

Reply:
They are available from `src/core/server/mocks`

Reply:
Thanks, fixed in https://github.com/elastic/kibana/pull/97334